src/ui/Card.tsx

```diff
@@ -187,7 +203,7 @@ export function Card(props: CardProps): JSX.Element {
         );
       }
     })();
-  }, [props.threadId]);
+  }, [props.threadId, props.onCardOffScreen]);
```

Comment:
React is complaining: useEffect is missing dependency "props".

Haven't spent any time debugging yet.